templates/telemetry/update/update.4.parquetmr.txt

```diff
@@ -16,6 +16,7 @@ message update {
       required binary architecture (UTF8);
       required binary version (UTF8);
       required binary buildId (UTF8);
+      required binary displayVersion (UTF8);
```

Comment:
You need to add a new `required group application` for this field - it comes from the application part of the submission [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/schemas/telemetry/update/update.4.schema.json#L4-L38).

That group should live at the same level as `metadata` or `environment`.

It should look something like
```
required group application {
  required binary displayVersion (UTF8);
}
```